test_requirements.txt

```diff
@@ -0,0 +1,5 @@
+coveralls
```

Comment:
you should start this with `-r requirements.txt` so someone preparing for development will get everything s/he needs with one target.